publications.md

```diff
@@ -1167,7 +1167,13 @@ Solving Once](https://www.semanticscholar.org/paper/Multiplex-Symbolic-Execution
   Lukáš Korenčik, Petr Ročkai  
   IEEE International Conference on Software Quality, Reliability and Security (QRS 2020)  
   Dezember 11-14, 2020, Macau, China
-
+ 
+1. [Automated Patch Transplantation](https://dl.acm.org/doi/10.1145/3412376)  
+  Ridwan Shariffdeen, Shin Hwei Tan, Mingyuan Gao, Abhik Roychoudhury
```

Comment:
You need to add two spaces at the end of this line and the next one, otherwise there is no line-break.

Reply:
Note: the spaces are still missing.

Reply:
Two more things:
1. Remove the spaces from the last line (the one with the link); sorry for the annoying markup format.
2. Could you please squash the commits ([HOWTO: squashing](https://github.com/openedx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#squash-your-changes=))?